prov/efa/src/rxr/rxr_msg.c

```diff
@@ -159,9 +116,6 @@ int rxr_msg_select_rtm(struct rxr_ep *rxr_ep, struct rxr_tx_entry *tx_entry, int
 		return RXR_EAGER_MSGRTM_PKT + tagged;
 	}
 
-	if (use_p2p && efa_mr_is_hmem(tx_entry->desc[0]))
```

Comment:
It seems we silently ignore use_p2p now. But it should be OK as we always require p2p for hmem so this cannot be false in that path.

Reply:
no action.